modules/nw-udn-examples.adoc

```diff
@@ -0,0 +1,69 @@
+//module included in the following assembly:
+//
+// *networking/multiple_networks/understanding-user-defined-networks.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="nw-udn-examples_{context}"]
+= Configuration details and examples of UserDefinedNetworks
+
+The following sections includes configuration details and examples  for creating user-defined networks (UDN) using the custom resource definition.
+
+[id=configuration-details-layer-two_{context}]
+== Configuration details for Layer2 topology
+The following applies when creating a UDN with a `Layer2` topology:
+
+* The `subnet` field is optional.
+* The `subnet` field is of type `string` and accepts standard CIDR formats for both IPv4 and IPv6.
+* The `subnets` field accepts one or two items. For two items, they must be of a different family. For example, `subnets` values of `10.100.0.0/16` and `2001:db8::/64`.
+* `Layer2` subnets may be omitted. If omitted, users must configure IP addresses for the pods. As a consequence, port security only prevents MAC spoofing.
+* The `Layer2` `subnets` field is mandatory when `ipamLifecycle` is specified.
+
+.Example of UDN over `Layer2` topology
+[%collapsible]
+====
+[source,terminal]
+----
+apiVersion: k8s.ovn.org/v1
+kind: UserDefinedNetwork
+metadata:
+  name: udn-network-primary
+  namespace: <example_namespace2>
+spec:
+  topology: Layer2
+  layer2:
+    role: Primary
+    mtu: 1300
+    subnets: ["10.150.0.0/16"]
+----
+====
+
+[id=configuration-details-layer-three_{context}]
+== Configuration details for Layer3 topology
+The following applies when creating a UDN with a `Layer3` topology:
+
+* The `subnets` field is mandatory.
+* The type for `subnets` field is `cidr` and `hostsubnet`:
++
+** `cidr` is the cluster subnet and accepts a string value.
+** `hostSubnet` specifies the nodes subnet prefix that the cluster subnet is split to.
+
+.Example of UDN over `Layer3` topology
+[%collapsible]
+====
+[source,terminal]
+----
+apiVersion: k8s.ovn.org/v1
+kind: UserDefinedNetwork
+metadata:
+  name: udn-network-primary
+  namespace: <example_namespace>
+spec:
+  topology: Layer3
+  layer3:
+    role: Primary
+    mtu: 1300
```

Comment:
tiny nit: I would keep it simple and not mention MTU.
One should know what they doing when changing it.

Reply:
removed